css-layout-classes/style.css

```diff
@@ -0,0 +1,31 @@
+img {
+  width: 100%;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
```

Comment:
The container will be a block element by default (meaning it will take up the full width). This would be considered an anti-pattern, because setting flex will make the child elements fit in one "row" and then you're setting it back to a "column" to get your layout to display properly.

You can just remove those properties here